### PR TITLE
allow 6 grid edbrand apps on homepage (bug 1140066)

### PR DESCRIPTION
**Pull request by spasovski**
I added the "getting around" listing on the "restofworld" homepage to easily test this on the -dev endpoint.

### CODE REVIEW

File: src/templates/_macros/feed_item.html [hunk 1]
```diff
@@ -55,7 +55,7 @@ <h1 class="feed-item-name">{{ feed_app.app.name|translate(feed_app) }}</h1>
 
 {% macro brand(brand) %}
   {% set brand = feed.transformer(brand) %}
-  {% set max_apps = 4 %}
+  {% set max_apps = 6 if brand.layout == 'grid' else 4 %}
```
Comment (ngokevin) [inline src/templates/_macros/feed_item.html]: i'd be clever and set this in the transformer in feed.js so you have ```brand.max_apps```

Comment (spasovski) [inline src/templates/_macros/feed_item.html]: Yea I agree just not used to the transformer yet. Consider it done.

**Comment by spasovski** [comment]
Updated.

**Comment by ngokevin** [comment]
retrying travis, a lot of failures

**Comment by ngokevin** [comment]
Think there are some legit failures for some reason ^

**Comment by spasovski** [comment]
Fixed tests.

**Pull request merged**